### PR TITLE
WIP: state Julia-i18n objectives.

### PR DESCRIPTION
**Pull request by Ismael-VC**
Please let me know if I have left something out and how can this be improved, thanks in advance!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
 # julia-i18n
 Julia language internationalization and localization project.
 
-# Roadmap
+## Objectives
+
+As stated in [#1](https://github.com/Julia-i18n/julia-i18n/issues/1), the goal and scope of this organization is to cover **all things i18n and l10n in Julia** (numeronims stand for *internationalization* and *localization* respectively). This includes but is not limmited to:
+
+* i18n and l10n of the official Julia website and other relevant julian websites, like the package listing website, bloggs, etc.
```
Comment (waldyrious) [inline README.md]: "bloggs" --> "blogs"

File: README.md [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
+* i18n and l10n of the official Julia [documentation](http://docs.julialang.org/en/latest/), [open books](https://en.wikibooks.org/wiki/Introducing_Julia) and manuals as well as other important sources of documentation like [*StackOverflow Documentation*](http://stackoverflow.com/documentation/julia-lang/topics), [*Learn Julia in Y minutes*](https://learnxinyminutes.com/docs/julia/), etc. whenever and however possible.
+* i18n and l10n of the documentation system in general, including the [documentation of Julia packages](https://github.com/JuliaDocs/Documenter.jl/issues/297), etc.
+* i18n and l10n of transcripts, captions, etc. in [video lectures](https://www.youtube.com/user/JuliaLanguage) and [video tutorials](https://www.youtube.com/watch?v=4nPmKG_f8-M&list=PLsu0TcgLDUiIznEhN165XmykqyLgzwY0Y), [podcasts](http://www.rce-cast.com/Podcast/rce-107-julia.html), etc.
+* And even the i18n and l10n of [Julia itself](https://groups.google.com/forum/#!topic/julia-dev/Vu6zyCPkkFs).
+
+If you happen to think Julia-i18n objectives are not [greedy](http://julialang.org/blog/2012/02/why-we-created-julia), please send us a pull request with your ungracious demands!
```
Comment (waldyrious) [inline README.md]: "Julia-i18n objectives" --> "Julia-i18n's objectives"

File: README.md [hunk 3]
```diff
@@ -1,8 +1,21 @@
 # julia-i18n
 Julia language internationalization and localization project.
 
-# Roadmap
+## Objectives
+
+As stated in [#1](https://github.com/Julia-i18n/julia-i18n/issues/1), the goal and scope of this organization is to cover **all things i18n and l10n in Julia** (numeronims stand for *internationalization* and *localization* respectively). This includes but is not limmited to:
```
Comment (ararslan) [inline README.md]: "numeronims" --> "numeronyms"

Comment (ararslan) [reply README.md]: "limmited" --> "limited"

File: README.md (hunk 1 above)
Comment (ararslan) [reply README.md]: Should capitalize "Julian"

File: README.md (hunk 3 above)
Comment (ararslan) [reply README.md]: Should probably be "numeronyms _which_ stand for ..."

File: README.md (hunk 1 above)
Comment (Ismael-VC) [reply README.md]: I always forget that! In Spanish, demonyms are never capitalized.

**Comment by Ismael-VC** [comment]
I have updated the PR with all your corrections, thanks for taking your time to review, I'll wait more time and see if more corrections or suggestions pop up.

**Comment by Ismael-VC** [comment]
I've updated with correction.

**Pull request merged**